dags/process_elife_zip_dag.py

```diff
@@ -212,6 +220,15 @@ def strip_related_article_tags_from_article_xml(**context) -> bytes:
     return etree.tostring(article_xml, xml_declaration=True, encoding='UTF-8')
 
 
+def add_missing_uri_schemes(**context) -> bytes:
+    article_xml = get_article_from_previous_task(context)
+    elements = article_xml.xpath('//*[starts-with(@xlink:href, "www.")]', namespaces=XLINK)
+    for element in elements:
+        element.attrib[XLINK_HREF] = 'http://' + element.attrib[XLINK_HREF]
```

Comment:
`http` is probably the safest default wrt `https`, it is usually redirected to `https` if available